### DISCUSSION TITLE
Dregsy sync between ECR Repos

### DISCUSSION THREAD

**Discussion by sydco08** [xelalexv/dregsy#84] [General]
Does dregsy support syncing images between two ECR repos in different accounts. If so how can this be accomplished.

was trying to do this using docker in github actions.
`          sudo docker run --privileged --rm -v /actions-runner/_work/Kubernetes/Kubernetes/.github/scripts/dregsync.yaml:/config.yaml \
          -v /var/run/docker.sock:/var/run/docker.sock \
          -e LOG_LEVEL=debug -e AWS_DEFAULT_REGION=$REGION \
          xelalex/dregsy`

however i get this error:
`time="2022-11-23T16:10:03Z" level=error msg="NoCredentialProviders: no valid providers in chain. Deprecated.\n\tFor verbose messaging see aws.Config.CredentialsChainVerboseErrors"`

i understand that it needs extra env variables to set the ACCESS_KEY and SECRET_ACCESS_KEY however i do not know how to set this for both the ECR repos.

**Comment by xelalexv** [comment]
This is unfortunately not yet possible. We need a more generalized config of AWS access credentials.
